telemetry/definitions/commonDefinitions.json

```
@@ -354,6 +354,16 @@
             "type": "string",
             "description": "The id of the experiment being activated or deactivated"
         },
+        {
+            "name": "settingId",
+            "type": "string",
+            "description": "The id of the setting being changed"
+        },
+        {
+            "name": "settingState",
+            "type": "string",
+            "description": "The state of the setting being changed to"
```

Generic metrics are more likely to be used for a wide variety of purposes, some of which we probably don't want. While this is unlikely to be problem, we'll want to establish the intended use right away just in case.

```suggestion
            "description": "The state of the setting being changed to. This should not be recorded for free-form settings like file-system paths. Instead, stick to things like flags, numbers, and enums."
```